frontend/app/projects/controller.js

```diff
@@ -147,7 +147,7 @@ export default class ProjectsController extends Controller {
   updateRemainingEffort(changeset) {
     if (
       changeset.get("project.remainingEffortTracking") &&
-      !changeset.get("mostRecentRemainingEffort")
+      this.selectedTask.isNew
```

Comment:
What is the value of `changeset.get("mostRecentRemainingEffort")` in case `this.selectedTask.isNew === true`?